### PR TITLE
Add telemetry to UI app

### PR DESCRIPTION
**Pull request by pritish-devurkar**
## Purpose
This PR adds application insights/telemetry to the UI app.
It also removes the search settings keys/values mistakenly checked in to the appsettings.json file
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Web UI/CognitiveSearch.UI/CognitiveSearch.UI.csproj [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <ApplicationInsightsResourceId>/subscriptions/789908e0-5fc2-4c4d-b5f5-9764b0d602b3/resourcegroups/knowledgemining-og-rg/providers/microsoft.insights/components/kminingdemoapp</ApplicationInsightsResourceId>
```
Comment (jainr) [inline Web UI/CognitiveSearch.UI/CognitiveSearch.UI.csproj:5]: do we need to put the sub-id and rg name here, it probably is needed by our deployed app but not here suggest removing it.

File: Web UI/CognitiveSearch.UI/appsettings.json [hunk 2]
```diff
@@ -5,37 +5,20 @@
       "Default": "Warning"
     }
   },
-
-    // Required fields
```
Comment (jainr) [inline Web UI/CognitiveSearch.UI/appsettings.json:9]: Please rotate the keys, and I think there is a way in git to get rid of a specific commit from the logs.

File: Web UI/CognitiveSearch.UI/appsettings.json [hunk 3]
```diff
@@ -5,37 +5,20 @@
...
-  // Optional key to an Azure Maps account if you would like to display the geoLocation field in a map
   "AzureMapsSubscriptionKey": "cz3FUHrarBCljsYSDzp57wRlIG93k4IddPAHIuRyZ7A",
-
-  // Set to the name of a facetable field you would like to represent as a graph.
-  // You may also set to a comma separated list of the facet names if you would like more than one facet type on the graph.
-  "GraphFacet": "keyPhrases, locations"
- 
+  "GraphFacet": "keyPhrases, locations",
+  "ApplicationInsights": {
+    "InstrumentationKey": "0b6b8876-422b-4b3c-9fff-314048b2880d"
```
Comment (jainr) [inline Web UI/CognitiveSearch.UI/appsettings.json:22]: Same with this key, please remove it and replace with a dummy to show as placeholder.

File: Web UI/CognitiveSearch.UI/CognitiveSearch.UI.csproj (hunk 1 above)
Comment (pritish-devurkar) [reply Web UI/CognitiveSearch.UI/CognitiveSearch.UI.csproj]: removed this, and replaced it with "_subsriptionId_" and "_resourceGroupName_"

File: Web UI/CognitiveSearch.UI/appsettings.json (hunk 2 above)
Comment (pritish-devurkar) [reply Web UI/CognitiveSearch.UI/appsettings.json:9]: Thanks, ran git revert and re-added back the changes.

File: Web UI/CognitiveSearch.UI/appsettings.json (hunk 3 above)
Comment (pritish-devurkar) [reply Web UI/CognitiveSearch.UI/appsettings.json]: Done, replaced with an empty string now.

**Pull request closed without merging**